boot/freeldr/freeldr/arch/i386/i386bug.c

```diff
@@ -115,7 +116,7 @@ i386PrintExceptionText(ULONG TrapIndex, PKTRAP_FRAME TrapFrame, PKSPECIAL_REGIST
     i386_ScreenPosX = 0;
     i386_ScreenPosY = 0;
 
-    PrintText("An error occured in " VERSION "\n"
+    PrintText("Freeldr " KERNEL_VERSION_STR " " KERNEL_VERSION_BUILD_STR "\n"
```

Comment:
```suggestion
    PrintText(VERSION " for ReactOS " KERNEL_VERSION_STR " " KERNEL_VERSION_BUILD_STR "\n"
```
This proposal would be more accurate, since it's a bsod in FreeLoader that is used to boot that version of ReactOS.
It would look like that:
![freeldr_bsod_hermes](https://github.com/user-attachments/assets/10176ac1-47c5-47d5-a863-705daa2b1185)